index.html

```diff
@@ -0,0 +1,1305 @@
+            </div>
+          </div>
+          <div class="card story-card mt-4 ">
+            <img
+              src="https://d.newsweek.com/en/full/1571844/bernie-sanders-joe-biden-missouri-primary-poll.jpg?w=397&amp;h=265&amp;l=49&amp;t=34&amp;q=90&amp;f=7b3fdc0e203af4625a2cc7333316e9c1"
+              alt="Bernie Sanders Joe Biden Missouri primary poll"
+              title="Bernie Sanders Joe Biden Missouri primary poll" class="mapping-sm_md card-img-top img-fluid">
+            <div class="card-body p-0">
+              <h5 class="card-title font-weight-bold pt-2">
+                Biden's Missouri Lead Over Sanders Fueled by Over-50s and Non-Whites: Poll
+              </h5>
+              <p class="card-text mb-0">The former vice president is ahead of the Vermont senator in Missouri, a poll
+                shows, but his lead is within the margin of error.</p>
+            </div>
+            <div class="card-footer bg-white pl-0 story-footer font-weight-bold">
+              Trump Says Sanders Would Be A 'Tougher' Opponent Than Frontrunner Joe Biden
+            </div>
+          </div>
+          <div class="card story-card mt-4 sticky-top">
+            <img
+              src="https://d.newsweek.com/en/full/1571844/bernie-sanders-joe-biden-missouri-primary-poll.jpg?w=397&amp;h=265&amp;l=49&amp;t=34&amp;q=90&amp;f=7b3fdc0e203af4625a2cc7333316e9c1"
+              alt="Bernie Sanders Joe Biden Missouri primary poll"
+              title="Bernie Sanders Joe Biden Missouri primary poll" class="mapping-sm_md card-img-top img-fluid">
+            <div class=" card-body p-0">
+              <h5 class="card-title font-weight-bold pt-2">
+                Biden's Missouri Lead Over Sanders Fueled by Over-50s and Non-Whites: Poll
+              </h5>
+              <p class="card-text mb-0">The former vice president is ahead of the Vermont senator in Missouri, a poll
+                shows, but his lead is within the margin of error.</p>
+            </div>
+            <div class="card-footer bg-white pl-0 story-footer font-weight-bold ">
+              Trump Says Sanders Would Be A 'Tougher' Opponent Than Frontrunner Joe Biden
+            </div>
+          </div>
+        
+      </div>
+
+
+      <div class="col-12 col-lg-6 col-md-6 ">
+        <div class="col-12">
+          <div class="row">
+            <h5 class="text-muted mt-4">TOP STORY</h5>
+            <div class="card pl-md-0">
+              <img
+                src="https://d.newsweek.com/en/full/1571844/bernie-sanders-joe-biden-missouri-primary-poll.jpg?w=397&amp;h=265&amp;l=49&amp;t=34&amp;q=90&amp;f=7b3fdc0e203af4625a2cc7333316e9c1"
+                alt="Bernie Sanders Joe Biden Missouri primary poll"
+                title="Bernie Sanders Joe Biden Missouri primary poll"
+                class="mapping-sm_md top-img card-img-top img-fluid">
+              <div class="card-body p-0">
+                <h5 class="card-title font-weight-bold pt-2">
```

Comment:
In the small breakpoint, this section should be the first one of the page, followed by the featured stories and then the opinion. You can achieve this using bootstrap classes.

Add image badges to the main section images.

Reply:
fixed